crpg.py

```diff
@@ -99,7 +100,7 @@ def generate(filename):
         for node in nodes.split("\n"):
             n, n_type, *args = re.split(r"\s*\|\s*", node)
 
-            if n_type == "fight" or n_type == "run":
+            if n_type == "fight" or n_type == "run" or n_type[:2] == "po":
```

Comment:
You should be able to specify n_type == "potion" here. n_type stands for node type, and is the second item in the .dl file

Reply:
Done.